### PR TITLE
contrib: added notes custom field

### LINKED ISSUE #1217: Implement notes field
Implement the field `notes`, which is a  `string`.

Take into account the following requirements when implementing this field:

- It is needed in zenodo-rdm. 
- It must be a custom field. 
- It must be searchable. Search must be compatible with the current [Zenodo search guide](https://help.zenodo.org/guides/search/)
- If needed, update DataCite serializer taking into account [Datacite Schema 4.4 docs](https://support.datacite.org/docs/datacite-metadata-schema-44). 
  - A recommended field [`RelatedItem` ](https://support.datacite.org/docs/datacite-metadata-schema-v44-recommended-and-optional-properties#20-relateditem) can be used to serialize journal's information.

### PR DESCRIPTION
**Pull request by alejandromumo**
closes #1217

### CODE REVIEW

File: invenio_rdm_records/contrib/notes/__init_.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+"""Notes contrib module."""
+from .custom_fields import (
```
Comment (alejandromumo) [inline invenio_rdm_records/contrib/notes/__init_.py:8]: I am not sure why this import is failing in tests.

Comment (ppanero) [reply invenio_rdm_records/contrib/notes/__init_.py:8]: This is because of the file name, it's missing a final `_` 👀

**Review by ppanero — APPROVED** [review]
Assuming tests pass

File: invenio_rdm_records/contrib/notes/__init_.py (hunk 1 above)
Comment (alejandromumo) [reply invenio_rdm_records/contrib/notes/__init_.py]: Thanks! Classic rookie mistake 😄

**Comment by alejandromumo** [comment]
Tests are failing due to [Babel](https://pypi.org/project/Babel/#history) new release. Not related with this PR

**Comment by alejandromumo** [comment]
There is no need to add a `notes` custom field. See this [discussion](https://github.com/inveniosoftware/invenio-rdm-records/issues/1217#issuecomment-1451649472).

Therefore, I am closing this PR without merging it.

**Pull request closed without merging**